docs/ibm-cloud-rules.md

```diff
@@ -1344,6 +1351,53 @@ components:
 </table>
 
 
+### Rule: operation-summary
+<table>
+<tr>
+<td><b>Rule id:</b></td>
+<td><b>operation-summary</b></td>
+</tr>
+<tr>
+<td valign=top><b>Description:</b></td>
+<td>This rule verifies that each operation has a non-empty.
+</td>
+</tr>
+<tr>
+<td><b>Severity:</b></td>
+<td>warn</td>
+</tr>
+<tr>
+<td><b>OAS Versions:</b></td>
+<td>oas3</td>
+</tr>
+<tr>
+<td valign=top><b>Non-compliant example:<b></td>
+<td>
+<pre>
+paths:
+  '/v1/things':
+    get:
+      operationId: list_things
+      description: Create a new Thing instance.
```

Comment:
I know it's just an example but "list" in the ID and "create"  in the description are at odds which either other. Same for the compliant example

Reply:
oops

Reply:
fixed